ex3.html

```diff
@@ -1,555 +1,558 @@
+										</select>
+									</li>
+									<li>Libraries
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>		
+										</select>
+									</li>
+									<li>Promises
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>				
+										</select>
+									</li>
+									<li>Browser Engines
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>			
+										</select>
+									</li>
+									<li>Frameworks
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>			
+										</select>
+									</li>
+								</ul>
+
+							</fieldset>
+
+
+
+							<fieldset>
+								<legend>NodeJS</legend>
+								<ul>
+									<li>Backend frameworks
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>	
+										</select>
+									</li>
+									<li>Templating
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>DOM Manipulation
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>Unit Testing
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>				
+										</select>
+									</li>
+								</ul>
+
+							</fieldset>
+
+							<h3><a href="ex2.html" target="_top" style="float: right;">SUBMIT</a></h3>
+						</form>
+
+
+						<style type="text/css">
+							table {
+								font-family: arial, sans-serif;
+								border-collapse: collapse;
+								width: 100%;
+							}
+
+							td, th {
+								border: 1px solid #dddddd;
+								text-align: left;
+								padding: 8px;
+							}
+
+							tr:nth-child(1) {
+								background-color: #dddddd;
+							}
+						</style>
+
+
+					</div>
+				</section>
+
+
+				<footer>
+					<h2>Copyright@Softvision 2017</h2>
```

Comment:
I'm still not too thrilled about using an `<h2>` tag here, in the page's footer, since it lacks semantic relevance...